go/consensus/tendermint/apps/staking/slashing.go

```diff
@@ -96,6 +97,13 @@ func onEvidenceDoubleSign(
 		)
 		return err
 	}
+	if totalSlashed.IsZero() {
```

Comment:
Note:
Before the `stakeState.SlashEscrow` would fail in case nothing was slashed (see other comment about `slashPool` method). So this keeps the logic consistent with what it was before.